music_album_management.rb

```diff
@@ -50,19 +49,45 @@ def list_genres
   @genres.each { |genre| puts "ID: #{genre.id}, Name: #{genre.name}" }
 end
 
-def collect_inputs
-  print 'What is the publish date(YYYY-MM-DD): '
-  date = gets.chomp
+def date_valid?(date)
+  format = '%Y-%m-%d'
```

Comment:
An invalid data here causes an error (for ex 2020-1-1)

![error](https://github.com/abel-tefera/catalog-of-my-things/assets/40035531/df0be501-6a70-4b2e-b53e-cd2b95020193)

Reply:
@abel-tefera It doesn't raise any error on my end. Not sure what is going on but we can discuss it in the afternoon.
<img width="953" alt="Screenshot 2023-07-05 at 10 15 49 AM" src="https://github.com/abel-tefera/catalog-of-my-things/assets/106236756/9c0ad6b9-f507-46ca-aeaf-14e478cfb423">

Reply:
@newhoteng Try to finish the interaction. After you choose the y/n for spotify, it will cause an error.

Reply:
@abel-tefera You were right, I've fixed it now. Can you please check again? Thanks